authentication/auth.go

```diff
@@ -95,102 +96,152 @@ func loadSigningKey() (*PrivateKeyResult, *KeyLoadError) {
 		Type:  "PUBLIC KEY",
 		Bytes: PublicKey,
 	})
-
 	kid := fmt.Sprintf("%x", sha1.Sum(pubBytes))
 
 	return &PrivateKeyResult{privateKey, kid}, nil
 }
 
-type eqClaims struct {
+// EqClaims is a representation of the set of values needed when generating a valid token
+type EqClaims struct {
 	jwt.Claims
-	UserID                string        `json:"user_id"`
-	EqID                  string        `json:"eq_id"`
-	PeriodID              string        `json:"period_id"`
-	PeriodStr             string        `json:"period_str"`
-	CollectionExerciseSid string        `json:"collection_exercise_sid"`
-	RuRef                 string        `json:"ru_ref"`
-	RuName                string        `json:"ru_name"`
-	RefPStartDate         string        `json:"ref_p_start_date"`         // iso_8601_date
-	RefPEndDate           string        `json:"ref_p_end_date,omitempty"` // iso_8601_date
-	FormType              string        `json:"form_type"`
-	SurveyURL             string        `json:"survey_url,omitempty"`
-	ReturnBy              string        `json:"return_by"`
-	TradAs                string        `json:"trad_as,omitempty"`
-	EmploymentDate        string        `json:"employment_date,omitempty"` // iso_8601_date
-	RegionCode            string        `json:"region_code,omitempty"`
-	LanguageCode          string        `json:"language_code,omitempty"`
-	VariantFlags          *variantFlags `json:"variant_flags,omitempty"`
-	Roles                 string        `json:"roles,omitempty"`
-	TxID                  string        `json:"tx_id,omitempty"`
+	UserID                string       `json:"user_id"`
+	EqID                  string       `json:"eq_id"`
+	PeriodID              string       `json:"period_id"`
+	PeriodStr             string       `json:"period_str"`
+	CollectionExerciseSid string       `json:"collection_exercise_sid"`
+	RuRef                 string       `json:"ru_ref"`
+	RuName                string       `json:"ru_name"`
+	RefPStartDate         string       `json:"ref_p_start_date"`         // iso_8601_date
+	RefPEndDate           string       `json:"ref_p_end_date,omitempty"` // iso_8601_date Optional
```

Comment:
Why do we need the comment for `Optional`? isn't this just shown by to `omitempty` flag?